lib/elixir/lib/calendar/date.ex

```diff
@@ -659,6 +659,106 @@ defmodule Date do
     calendar.day_of_week(year, month, day)
   end
 
+  @doc """
+  Calculates the day of the year of a given `date`.
+
+  Returns the day of the year as an integer. For the ISO 8601
+  calendar (the default), it is an integer from 1 to 366.
```

Comment:
how about just the following?

```
For the ISO calendar (the default), it is an integer from 1 to 366.
```

Reply:
Thanks for the feedback,  makes sense for the @specs since they are calendar specific.